website/docs/r/redis_enterprise_database.html.markdown

```diff
@@ -50,8 +69,18 @@ The following arguments are supported:
 
 * `eviction_policy` - (Optional) Redis eviction policy - default is VolatileLRU. Possible values are `AllKeysLFU`, `AllKeysLRU`, `AllKeysRandom`, `VolatileLRU`, `VolatileLFU`, `VolatileTTL`, `VolatileRandom` and `NoEviction`. Defaults to `VolatileLRU`. Changing this forces a new Redis Enterprise Database to be created.
 
+-> **NOTE:** Eviction Policy must be set to NoEviction when using RediSearch module
+
 * `module` - (Optional)  A `module` block as defined below.
 
+-> **NOTE:** Only RediSearch module is allowed with geo-replication
+
+* `linked_database_id` - (Optional) A list of database resources to link with this database with a maximum of 5.
```

Comment:
so this should be force now via a customize diff?

Reply:
The linked database list can be changed by adding a new one/ removing an existing one, so cannot be made as forceNew. please bear with me if my understanding of the comment is incorrect. :)